<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recruitermenu</class>
 <widget class="QMainWindow" name="Recruitermenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main Window Background */
QWidget {
    background-color: #121212;
    color: #e0e0e0;
    font-family: 'Poppins', 'Segoe UI', sans-serif;
    font-size: 15px;
    letter-spacing: 0.4px;
    padding: 10px;
}

/* Labels */
QLabel {
    font-size: 17px;
    color: #ffffff;
    font-weight: 500;
}

/* Buttons */
QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #ff5c93, stop:1 #ffc371);
    border: none;
    border-radius: 14px;
    color: white;
    font-size: 15px;
    font-weight: bold;
    padding: 10px 22px;
    min-width: 120px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #ffc371, stop:1 #ff5c93);

}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #e34e82, stop:1 #ff9f57);

}

/* ComboBox */
QComboBox {
    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    border-radius: 10px;
    color: white;
    padding: 6px 12px;
    font-size: 14px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 18px;
    background-color: #ff5c93;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}

QComboBox QAbstractItemView {
    background-color: #1f1f1f;
    color: #ffffff;
    selection-background-color: #ff5c93;
    border-radius: 8px;
}

/* LineEdit */
QLineEdit {
    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    border-radius: 10px;
    color: white;
    font-size: 14px;
    padding: 8px 12px;
}

/* Checkboxes */
QCheckBox {
    color: #ffffff;
    font-size: 14px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border: 2px solid #ff5c93;
    border-radius: 4px;
    background-color: transparent;

}

QCheckBox::indicator:checked {
    background-color: #ff5c93;
    border: 2px solid #ff9f57;
}

/* ScrollBar */
QScrollBar:vertical {
    background: #1f1f1f;
    width: 12px;
    margin: 4px;
    border-radius: 6px;
}

QScrollBar::handle:vertical {
    background: #ff5c93;
    border-radius: 6px;
    min-height: 20px;
}

QScrollBar::add-line, QScrollBar::sub-line {
    background: none;
}

/* Table Widget */
QTableWidget {
    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    color: #ffffff;
    font-size: 14px;
    gridline-color: #ff5c93;
}

QHeaderView::section {
    background-color: #ff5c93;
    color: white;
    padding: 6px;
    border: none;
}

/* Tooltips */
QToolTip {
    background-color: #ff5c93;
    color: white;
    padding: 6px;
    border-radius: 8px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="skillInput">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultsTable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>701</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>164</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SEARCH SKILL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_companyName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>501</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    qproperty-alignment: 'AlignCenter'; /* Center-aligns the text */
f;
    font-size: 30px;
}
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
